api/tools/build.gradle

```diff
@@ -11,7 +11,7 @@ def workbench_db_password = System.getenv("WORKBENCH_DB_PASSWORD")
 
 def dbProperties = [
   'spring.datasource.driver-class-name': 'com.mysql.jdbc.Driver',
-  'spring.datasource.url': 'jdbc:mysql://${db_host}:${db_port}/workbench',
+  'spring.datasource.url': 'jdbc:mysql://${db_host}:${db_port}/workbench?useSSL=false',
```

Comment:
What is this change for?

Reply:
Reduces log noise and makes this consistent with the other one.

Reply:
Also, our database isn't configured to allow outside connections (which is good), so our connections are either local or through the proxy, which encrypts all traffic.

Reply:
Based on your question, I decided this was worth a comment.